### PR TITLE
EHN: adding isnan method in paddle.tensor.tensor

### LINKED ISSUE #17123: isnan
adding  isnan in paddle.tensor.tesnor.tesnor from issue #15115

### PR DESCRIPTION
**Pull request by MuhammadNizamani**
adding isnan from issue closed  #17123

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by MuhammadNizamani** [comment]
> If you are working on an open task, please **edit** the PR description to link to the issue you've created.
> 
> For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).
> 
> Thank you hugs
Sir I have mentioned the issue

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by aparajith21** [comment]
close #17123

**Comment by aparajith21** [comment]
All looks good! Thanks for contributing, could you push again after resolving conflicts and then can merge!

**Comment by MuhammadNizamani** [comment]
@aparajith21  I have resolve the conflict.

**Comment by aparajith21** [comment]
LGTM! Thank you for contributing!

**Comment by MuhammadNizamani** [comment]
add_frontend_checklist

**Comment by MuhammadNizamani** [comment]
@aparajith21  I have add check_list .

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 1]
```diff
@@ -274,6 +274,13 @@ def astype(self, dtype):
     def logical_xor(self, y, out=None, name=None):
         return paddle_frontend.logical_xor(self, y, out=out)
 
+    @with_supported_dtypes(
+        {"2.5.0 and below": ("float16", "float32", "float64", "int32", "int64")},
+        "paddle",
+    )
+    def isnan(self, name=None):
+        return ivy.isnan(self._ivy_array)
```
Comment (aparajith21) [inline ivy/functional/frontends/paddle/tensor/tensor.py:282]: Could you please change this to `paddle_frontent.isnan()`

Comment (MuhammadNizamani) [reply ivy/functional/frontends/paddle/tensor/tensor.py:282]: sure

Comment (MuhammadNizamani) [reply ivy/functional/frontends/paddle/tensor/tensor.py]: @aparajith21 done

**Comment by aparajith21** [comment]
LGTM, thanks for contributing!

**Pull request merged**